process/project-lifecycle-documents/openssf_scorecard_incubating_stage.md

```diff
@@ -0,0 +1,89 @@
+# OpenSSF Scorecard — Incubating application
+
+## Project has met all Sandbox requirements
+
+The only Sandbox application requirement that is not listed as part of the Incubation application superset is the matter of project sponsorship.
+
+### Sponsor
+
+Most projects will report to an existing OpenSSF Working Group, although in some cases a project may report directly to the TAC. The project commits to providing quarterly updates on progress to the group they report to.
+
+OpenSSF Scorecard is a project of the Best Practices Working Group.
+
+## List of project maintainers
+
+The project must have a minimum of three maintainers with a minimum of two different organizational affiliations.
+
+- Stephen Augustus, Cisco, [@justaugustus](https://github.com/justaugustus)
+- Raghav Kaul, Google, [@raghavkaul](https://github.com/raghavkaul)
+- Jeff Mendoza, Kusari, [@jeffmendoza](https://github.com/jeffmendoza)
+- Spencer Schrock, Google, [@spencerschrock](https://github.com/spencerschrock)
+- Laurent Simon, Independent, [@laurentsimon](https://github.com/laurentsimon)
+- Naveen Srinivasan, Independent, [@naveensrinivasan](https://github.com/naveensrinivasan)
+
+The current list of OpenSSF Scorecard maintainers can be found [here](https://github.com/ossf/scorecard/blob/main/MAINTAINERS.md).
+
+## Mission of the project
+
+The project must be aligned with the OpenSSF mission and either be a novel approach for existing areas, address an unfulfilled need, or be code needed to deliver OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
+
+The mission of OpenSSF Scorecard is to automate analysis on the security posture of open source projects.
+
+The current charter of the OpenSSF Scorecard project can be found [here](https://github.com/ossf/scorecard/blob/main/CHARTER.md).
```

Comment:
Please, don't use "here" as a link anchor. That's an anti-pattern (and a pet peeve of mine: https://lehors.wordpress.com/2009/01/29/linking-the-proper-way/ :-)